rcgen.js

```diff
@@ -1,23 +1,30 @@
 // @ts-check
 
-const {git, gitIgnore} = require('@rcgen/configs');
+const {
+  git,
+  gitIgnoreFiles,
+  gitIgnoreIntrinsicFiles,
+  node,
+  npm
+} = require('@rcgen/configs');
 const {composeManifest} = require('@rcgen/core');
 
 exports.default = composeManifest(
   git(),
-  gitIgnore({
-    additionalFilenames: [
-      '.cache',
-      'coverage',
-      'lerna-debug.log',
-      'lib',
-      'node_modules',
-      'npm-debug.log',
-      'package-lock.json',
-      'packages/website/build',
-      'packages/website/i18n',
-      'todo.tasks',
-      'yarn-error.log'
-    ]
-  })
+  gitIgnoreFiles(
+    '.cache',
+    'coverage',
+    'lerna-debug.log',
+    'lib',
+    'node_modules',
+    'npm-debug.log',
+    'package-lock.json',
+    'packages/website/build',
+    'packages/website/i18n',
+    'todo.tasks',
+    'yarn-error.log'
+  ),
+  gitIgnoreIntrinsicFiles({excludedFilenamePatterns: ['package.json']}),
```

Comment:
I don't understand why the `package.json` would be ignored by git "intrinsically".

Reply:
Because `npm()` is included two lines below and defines the `package.json` file. Currently the `package.json` file is already generated. This means read in but not yet modified, because there is no config provider there who wants to change something about it.

Reply:
Perhaps the term "intrinsic" is inappropriate? I found it better than e.g. "internal".